packages/devtools_app/lib/src/shared/console/widgets/evaluate.dart

```diff
@@ -48,6 +47,8 @@ class ExpressionEvalFieldState extends State<ExpressionEvalField>
 
   final evalTextFieldKey = GlobalKey(debugLabel: 'evalTextFieldKey');
 
+  final _evalService = EvalService();
```

Comment:
Seems like there needs to be a global instances of the EvalService rather than creating one with each EvalField. Creating one with each eval field would cause problems later when you start adding extra evalService functionality for injecting `$1, $_, etc`